mne/epochs.py

```diff
@@ -308,8 +318,64 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             else:
                 self.drop_log = drop_log
             events = events[selected]
-            if len(np.unique(events[:, 0])) != len(events):
-                raise RuntimeError('Event time samples were not unique')
+            u_evs, u_idxs, counts = np.unique(events[:, 0],
+                                              return_index=True,
+                                              return_counts=True)
+            if len(u_evs) != len(events):
+                msg = 'Multiple event codes for single event times found.'
+                if event_repeated == 'error':
+                    raise RuntimeError('Event time samples were not unique')
+
+                elif event_repeated == 'drop':
+                    logger.info('{} Keeping the first occurrence and '
+                                'dropping all others.'.format(msg))
+                    events = events[u_idxs]
+
+                elif event_repeated == 'merge':
+                    logger.info('{} Creating a new event code to reflect these'
+                                'simultaneous events and updating event_id.'
+                                .format(msg))
+                    new_events = events.copy()
+                    non_u_evs = u_evs[counts > 1]
+                    for ev in non_u_evs:
+
+                        # indices at which the non-unique events happened
+                        idxs = (events[:, 0] == ev).nonzero()[0]
+
+                        # Figure out event_codes prior to events to be merged
+                        prior_codes = events[idxs, 1]
```

Comment:
what you call event_codes is what we call event_id in the rest of the repo. I would suggest sticking to the conventions of the repo as much as possible and bootstrapping the names from there

Reply:
nope, event_codes are the VALUES ... if you have `event_id = {'my_stim': 1}`. then I see "my_stim" as the event_id and "1" as the event_code.

What nomenclature should I use instead? `event_id` and `event_value`?

Reply:
I am using `event_vals` instead of `event_codes` now ... and `prior_codes` is dropped completely :-)

that way should be better because we can draw on the key versus value distinction and in `event_id` we have the description as a key and the "TTL trigger code" (=event-marker) as a value